File: psup assignment/cpp/q12.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int marks;
    std::cout << "\n--- Program 12: Grade Classifier ---\n";
    std::cout << "Enter marks: ";
    std::cin >> marks;
    if (marks >= 90) {
        std::cout << "A" << std::endl;
    } else if (marks >= 75) {
        std::cout << "B" << std::endl;
    } else if (marks >= 50) {
        std::cout << "C" << std::endl;
    } else {
        std::cout << "Fail" << std::endl;
    }
    return 0;
}

File: psup assignment/java/q12.java
import java.util.*;
class GradeClassifier {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter marks: ");
        int marks = input.nextInt();
        if (marks >= 90) {
            System.out.println("A");
        } else if (marks >= 75) {
            System.out.println("B");
        } else if (marks >= 50) {
            System.out.println("C");
        } else {
            System.out.println("Fail");
        }
        input.close();
    }
}